### PR TITLE
basic README format

### PR DESCRIPTION
**Pull request by EvanM27**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bug fix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- README.md contains only one line

## What is the new behavior?

- README.md now contains useful information about who developed the application, technologies used in the app, and how to run the app on a local machine.

## Other information

READMEdemo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+- [General info](#general-info)
+- [Technologies](#technologies)
+- [Setup](#setup)
+
+## General info
+
+This project is the front end for the mount recorder application.
+It displays users, mounts, and which mounts users own.
+It hosts a variety of reporting options for all the above categories.
+It comes with a detailed help section with instructions on the apps use.
```
Comment (sofia819) [inline README.md]: Format with bullet points?

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+## Technologies
+
+Project is created with:
+
+- "node": "12.8.1"
+- "npm": "6.14.5"
+- "@material-ui/core": "^4.11.0"
+- "react": "^16.13.1"
+- "wretch": "^1.7.2"
```
Comment (sofia819) [inline README.md]: I'd probably put JS/React as language and framework and then MUI and wretch for libraries used.

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+## Setup
+
+To run this project, install it locally using npm:
+
+```
+$ cd ../mount_recorder_client
+$ npm install
+$ npm start
```
Comment (sofia819) [inline README.md]: Do a bullet point list here:
1. Clone the repository
2. Install dependencies using `npm install`
3. Start the server using `npm start`

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+```
+
+This application requires a backend to function as intended,
+with the purpose built back end located in this repository:
+[Mount Recorder Server](https://github.com/sofia819/mount_recorder_server)
+
+The backend can be replaced as long as the middleware calls are updated to
+match your backend.
```
Comment (sofia819) [inline README.md]: Unsure what this means. I'd remove this.

File: README.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+
+## Table of contents
+
+- [General info](#general-info)
+- [Technologies](#technologies)
+- [Setup](#setup)
+
+## General info
+
+This project is the front end for the mount recorder application.
```
Comment (sofia819) [inline README.md]: `front-end`?

**Review by sofia819 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (EvanM27) [reply README.md]: bullet point formatting added

File: README.md (hunk 5 above)
Comment (EvanM27) [reply README.md]: corrected

File: README.md (hunk 4 above)
Comment (EvanM27) [reply README.md]: removed

File: README.md (hunk 3 above)
Comment (EvanM27) [reply README.md]: changed to bullet point list

File: README.md (hunk 2 above)
Comment (EvanM27) [reply README.md]: Updated

**Pull request merged**